demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,145 @@
+
+## Building Python* Model API package
+For installation Python (version 3.6 or higher) is required. The installation is available from source.
+
+Use the following command to install Python* Model API from source:
+```sh
+pip install <omz_dir>/demos/common/python
+```
+
+Alternatively, you can generate the package using a wheel. Follow the steps below:
+1. Build the wheel.
+
+```sh
+python <omz_dir>/demos/common/python/setup.py bdist_wheel
+```
+The wheel should appear in the dist folder.
+Name example: `openmodelzoo_modelapi-0.0.0-py3-none-any.whl`
+
+2. Install the package in the clean environment with `--force-reinstall` key.
+```sh
+pip install openmodelzoo_modelapi-0.0.0-py3-none-any.whl --force-reinstall
+```
+
+To verify the package is installed, you might use the following command:
+```sh
+python -c "from openvino.model_zoo import model_api"
+```
+
+> **NOTE**: On Linux and macOS, you may need to type `python3` instead of `python`. You may also need to [install pip](https://pip.pypa.io/en/stable/installation/).
+> For example, on Ubuntu execute the following command to get pip installed: `sudo apt install python3-pip`.
+
+## Model API Wrappers
+
+The Python* Model API package suggests ready-to-use model wrappers, which implement standardized preprocessing/postprocessing functions per "task type" and might be reused in applications as "black-box" models.
+Also, the simple wrapper interface allows the creation of custom wrappers covering different architectures.
+
+The following tasks can be solved with wrappers usage:
+
+| Task type                  | Model API wrappers |
+|----------------------------|--------------------|
+| Background Matting         | `VideoBackgroundMatting`, `ImageMattingWithBackground` |
+| Classification             | `Classification` |
+| Deblurring                 | `Deblurring` |
+| Human Pose Estimation      | `HpeAssociativeEmbedding`, `OpenPose` |
+| Instance Segmentation      | `MaskRCNNModel`, `YolactModel` |
+| Monocular Depth Estimation | `MonoDepthModel` |
+| Named Entity Recognition   | `BertNamedEntityRecognition` |
+|  Object Detection          | `CenterNet`, `DETR`, `CTPN`, `FaceBoxes`, `RetinaFace`, `RetinaFacePyTorch`, `SSD`, `UltraLightweightFaceDetection`, `YOLO`, `YoloV3ONNX`, `YoloV4`, `YOLOF`, `YOLOX` |
+| Question Answering         | `BertQuestionAnswering` |
+| Salient Object Detection   | `SalientObjectDetectionModel` |
+| Semantic Segmentation      | `SegmentationModel` |
+
+## Model API Adapters
+
+Model API wrappers are executor-agnostic, meaning it does not implement the specific model inference or model loading, instead it can be used with different executors having the implementation of common interface methods in adapter class respectively.
+
+Currently, `OpenvinoAdapter` and `OVMSAdapter` are supported.
+
+#### OpenVINO executor
```

Comment:
probably better name them adapters, looks misleading if you do not provide what is executor in your terms, but call class of adapter

Reply:
Renamed